### PR TITLE
Make environments configurable

### PR DESCRIPTION
**Pull request by arnested**

### CODE REVIEW

File: bin/entrypoint [hunk 1]
```diff
@@ -4,14 +4,18 @@ set -e
 
 if test -z "$PROJECT"; then
     echo "\$PROJECT is empty"
-    exit 1
+#    exit 1
```
Comment (achton) [inline bin/entrypoint:7]: Remove or do not. There is no maybe.

**Review by achton — CHANGES_REQUESTED** [review]

File: bin/entrypoint (hunk 1 above)
Comment (arnested) [reply bin/entrypoint:7]: Tak. det er debug left over...

Comment (arnested) [reply bin/entrypoint]: Fixed

**Review by achton — APPROVED** [review]
LGTM! :+1:

**Pull request merged**